openbb_terminal/forex/forex_controller.py

```diff
@@ -77,7 +93,6 @@ def print_help(self):
         mt.add_param("_source", FOREX_SOURCES[self.source])
         mt.add_raw("\n")
         mt.add_cmd("quote", self.fx_pair)
-        mt.add_cmd("load", self.fx_pair)
```

Comment:
Is there a reason to delete the `load` method from this `ForexController` ?

Reply:
There was two `load` methods in the menu, so I deleted the later one.